File: ARCADE/The Core/Level 3_2: arrayPacking.cs
    using System;
 
public class Test
{
	public static int arrayPacking(int[] a) {
	    int rs = 0;
	    for(int i = 0; i < a.Length; i++){
	        rs += a[i] << (8 * i);
	    }
	    return rs;
	} 
	public static void Main()
	{
		int [] a = new int []{24, 85, 0};
		Console.WriteLine(arrayPacking(a));
	}
}

File: ARCADE/The Core/Level 3_2: arrayPacking.pas
program arrayPacking;
Type
    ArrayInteger = Array of integer;
function arrayPacking(a: ArrayInteger): integer;
var rs, i : integer;
begin
    rs := 0;
    for i := 0 to length(a) - 1 do
        rs := rs + a[i] shl (8 * i);
    arrayPacking := rs;
end;

var a : ArrayInteger;
begin
	a := ArrayInteger.Create(24, 85, 0);
	writeln(arrayPacking(a));
end.
